oxidation/crates/parsec_api_protocol/src/cmds.rs

```diff
@@ -0,0 +1,202 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+macro_rules! cmds_bundle {
```

Comment:
Can you add a comment (typically showing the load/dumps code for client/server side) to explain the rational behind this weird nested modules ? ;-)

Reply:
Weird ? :joy: Ok :+1:

Reply:
Done :+1: